src/config/inversion-fixes.config

```diff
@@ -1,7 +1,6 @@
 *
 
 INVERT
-img
```

Comment:
This should be a separate PR, but would break a lot of current setups/configurations